app/javascript/react/components/main.js

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+import ReactDom from "react-dom"
+
+import App from "./App"
+
+ReactDom.render
+  (<App />, 
+  document.getElementById("app")
+)
```

Comment:
This whole file can be deleted as we instead have a file in `app/javascript/packages/application.js` that will act as the pseudo `main.js`

Reply:
this was gone after re-pulling